osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -90,6 +96,15 @@ public OsuDifficultyHitObject(HitObject hitObject, HitObject lastObject, HitObje
             // Capped to 25ms to prevent difficulty calculation breaking from simultaneous objects.
             StrainTime = Math.Max(DeltaTime, min_delta_time);
 
+            if (BaseObject is Slider sliderObject)
+            {
+                HitWindowGreat = 2 * sliderObject.HeadCircle.HitWindows.WindowFor(HitResult.Great) / clockRate;
+            }
```

Comment:
I'm not too fond of doing this but it seems necessary since `Slider` hit windows are empty - their hit windows would be `0` otherwise. Is this the optimal way this can be done right now?

Reply:
I think this is fine.